src/zen/forks/fork8_sidechainfork.cpp

```diff
@@ -4,9 +4,9 @@ namespace zen {
 
 SidechainFork::SidechainFork()
 {
-    setHeightMap({{CBaseChainParams::Network::MAIN,999999}, // TODO set it properly
+    setHeightMap({{CBaseChainParams::Network::MAIN,1111111}, // TODO set it properly
                   {CBaseChainParams::Network::REGTEST,420},
-                  {CBaseChainParams::Network::TESTNET,750000}}); //MOVED UP TODO set it properly
+                  {CBaseChainParams::Network::TESTNET,900000}}); // TODO set it properly
```

Comment:
These preliminary fork heights differ from the heights added to test_forkmanager.cpp in https://github.com/HorizenOfficial/zend_oo/pull/144/files?file-filters%5B%5D=.cpp#diff-63ec16fb4d33e614d2b0bb4e8d0cf3d6b414a3e72cebf2933264f603facc24e0

Heights on #144 should be adjusted to the same heights as on this PR.